tests/utilities/isRegisteredProperty.js

```diff
@@ -10,139 +45,68 @@ export default {
 		document.body.append(iframe);
 
 		this.data.iframe = iframe;
-	},
 
-	run (property) {
-		if (typeof property === "object") {
-			property = "--" + property.id;
+		if (this.data.srcdoc !== undefined) {
+			iframe.srcdoc = this.data.srcdoc;
 		}
 
+		return new Promise(resolve => (iframe.onload = resolve));
+	},
+
+	run (property) {
 		return isRegisteredProperty(property, this.data.iframe.contentWindow);
```

Comment:
```suggestion
		return new Promise(resolve => {
			requestAnimationFrame(() => {
				resolve(isRegisteredProperty(property, this.data.iframe.contentWindow));
			});
		});
```

Reply:
It looks like we're facing a race condition issue in Safari. That's why some tests might fail in one run and pass in the next. This fixes it. Probably you can find another solution. Just wanted to share my findings with you.

By the way, we might face the same issue in our other tests. What if we extract this (or another snippet solving the problem) into a helper?